synapse/storage/schema/main/delta/69/01device_list_oubound_by_room.sql

```diff
@@ -14,15 +14,15 @@
  */
 
 CREATE TABLE device_lists_changes_in_room (
-    stream_id BIGINT NOT NULL,
-    room_id TEXT NOT NULL,
     user_id TEXT NOT NULL,
     device_id TEXT NOT NULL,
+    room_id TEXT NOT NULL,
+    stream_id BIGINT NOT NULL,  -- This matches `device_lists_stream.stream_id`
```

Comment:
> -- This matches `device_lists_stream.stream_id`

can we enforce this with a constraint?

Reply:
Annoyingly we delete from `device_lists_stream` the next time the user updates their device, and I *think* we'll want to keep the rows in this table when that happens.